blockmanager.go

```diff
@@ -98,12 +99,39 @@ type txMsg struct {
 	peer *ServerPeer
 }
 
+// blockManagerCfg holds options and dependencies needed by the blockManager
+// during operation.
+type blockManagerCfg struct {
+	// ChainParams is the chain that we're running on.
+	ChainParams chaincfg.Params
+
+	BlockHeaders     headerfs.BlockHeaderStore
+	RegFilterHeaders *headerfs.FilterHeaderStore
```

Comment:
Why is the block header store a regular value, while this is a pointer?

Reply:
One is a interface, the other not.